docs/source/development/traffic_router/traffic_router_api.rst

```diff
@@ -471,3 +471,65 @@ Request Structure
+
+Request Structure
+-----------------
+.. table:: Request Query Parameters
+
+	+-------------------+----------+--------------------------------------------------------------------------------------------------------------+
+	| Name              | Required | Description                                                                                                  |
+	+===================+==========+==============================================================================================================+
+	| requestPath       | yes      | The (URI encoded) request path to use to test pattern based consistent hashing                               |
+	+-------------------+----------+--------------------------------------------------------------------------------------------------------------+
+	| deliveryServiceId | yes      | The integral, unique identifier?/'xml_id'?/name? of a :term:`Delivery Service` served by this Traffic Router |
+	+-------------------+----------+--------------------------------------------------------------------------------------------------------------+
+
+Response Structure
+------------------
+TBD
+
+``/crs/consistenthash/cache/coveragezone/steering``
+===================================================
+The resulting cache of the consistent hash using coverage zone for a given client IP, delivery service and, request path -- used to test cache selection for steering delivery services.
+
+Request Structure
+-----------------
+.. table:: Request Query Parameters
+
+	+-------------------+----------+--------------------------------------------------------------------------------------------------------------+
+	| Name              | Required | Description                                                                                                  |
+	+===================+==========+==============================================================================================================+
+	| requestPath       | yes      | The (URI encoded) request path to use to test pattern based consistent hashing                               |
+	+-------------------+----------+--------------------------------------------------------------------------------------------------------------+
+	| deliveryServiceId | yes      | The integral, unique identifier?/'xml_id'?/name? of a :term:`Delivery Service` served by this Traffic Router |
+	+-------------------+----------+--------------------------------------------------------------------------------------------------------------+
+	| ip                | yes      | The IP address of a potential client                                                                         |
+	+-------------------+----------+--------------------------------------------------------------------------------------------------------------+
+
+Response Structure
+------------------
+TBD
```

Comment:
"TBD" in not an acceptable response structure (or response example). It should minimally contain an example - though the routes that omit structure explanations only do so because I don't understand them.

Reply:
I've included request/response structures for 2/3 of the routes I've added. I don't currently have the environment to get a successful response for the third (healthy caches), but I can add it at a later time